src/components/recipe-form.js

```diff
@@ -131,7 +141,7 @@ const UnitDropdown = props => (
 const RecipeForm = ({
   recipe_id = null,
   name = '',
-  ingredients = [{ name: '', unit: '', amount: '' }],
+  ingredients = [{ name: '', unit: '', amount: 0 }],
```

Comment:
why are we defaulting it to 0? Don't we want it to be blank at first?

Reply:
you gotta point. The result is the same due to some code later, but it should be '' yee